stories/profile/PriorityBillsCard.stories.tsx

```diff
@@ -24,12 +21,188 @@ const Template: ComponentStory<typeof PriorityBillsCard> = props => {
   props.onClick = (string: string) => onBillSelected(string)
   props.selectedBillId = selectedBillId
 
-  console.log("rerender", props)
   return <PriorityBillsCard {...props} />
 }
 
+//editBtn: true,
 export const Primary = Template.bind({})
 Primary.args = {
+  bills: [
```

Comment:
let's deduplicate things some and move these to constants shared between the stories